### ISSUE TITLE
Awesome scripts, I loved them!

### ISSUE THREAD

**Issue by jmiahman** [dillj/Unity-Linux#1]
Your scripts were awesome and I would like to integrate them and have them replace my crappy build scripts at some point. If at any point in time you feel like doing so let me know. I use git now instead of svn and I am willing to change things around to make things easier. I just need to know what. If you're interested in working with me let me know. Thanks

Kind regards

JMiahMan

**Comment by mdawkins** [comment]
Which ones precisely? You know I'll help with what ever I can.

Matt

**Comment by jmiahman** [comment]
Hey now.. I'll have to take a look. I was going to start piece mealing them myself but then I noticed 18 days ago Jon Dill forked the Unity repo so I thought well maybe he would be interested.

**Comment by mdawkins** [comment]
Yeah, he did it to save what we had left after yoper's svn/ hosting took a
dive. Depends on what you like, the ISO build lists are pretty much much
mine. The bldpkg bldchrt is mine and stumpys . other stuff might be kds or
Paul's...

Matt

**Comment by jmiahman** [comment]
Yeah I am using really crappy scripts, though I have my own init and makelive script I would like to improve and use instead of mklivecd. It's all very rough right now, but I would like to get some of the build scripts and rpm tools we had going again as in a sense that was a identity we had and something we could explain and document for new users pretty easily and they made tracking and development easier.

**Comment by mdawkins** [comment]
OK, the problem I see ATM is pkg name translation, but it's easily over
come....

Do you have a base list yet?

Matt

**Comment by dillj** [comment]
Hi, yeah as Matt mentioned, I simply put our stuff on git as a backup. 
It's been a while since I've done anything unity-linux related..

**Comment by jmiahman** [comment]
mdawkins.. yes I may need to do some tweaking of it

musl
busybox
unity-baselayout
openrc
unity-conf
zlib
libcrypto.10
libssl1.0
rpm
yum
busybox-initscripts
scanelf
musl-utils

These packages should pull in other packages that are needed if you use yum. Like rpm5 needs beecrypt, db5.2, popt etc..
